packages/basic/lib/worker/child-manager.js

```diff
@@ -165,6 +167,14 @@ export class ChildManager extends ITC {
     )
 
     process.env.PLT_MANAGER_ID = this.#id
+
+    if (this.#context.telemetryConfig) {
+      const require = createRequire(import.meta.url)
+      const telemetryPath = require.resolve('@platformatic/telemetry')
+      const openTelemetrySetupPath = join(telemetryPath, '..', 'lib', 'node-http-telemetry.js')
+      process.env.NODE_OPTIONS =
```

Comment:
This won't work: in this way `node-http-telemetry.js` will be executed after `./child-process.js`.

Either you move the if block after line 178 or you build the NODE_OPTIONS in a more dynamic way, like:

```
const nodeOptions = process.env.NODE_OPTIONS ?? ''
const childProcessInclude = `--import="${new URL('./child-process.js', import.meta.url)}"`
let telemetryInclude = ''

if (this.#context.telemetryConfig) {
  const require = createRequire(import.meta.url)
  const telemetryPath = require.resolve('@platformatic/telemetry')
  const openTelemetrySetupPath = join(telemetryPath, '..', 'lib', 'node-http-telemetry.js')

  telemetryInclude = `--require="${openTelemetrySetupPath}"`
}

process.env.NODE_OPTIONS = `${telemetryInclude} ${childProcessInclude} ${nodeOptions}`.trim()
```

Reply:
Correct, good spot, tnx :)